index.html

```diff
@@ -1115,6 +1129,36 @@
             ctx.fillRect(0, urlBarOffset, urlBarWidth, urlBarHeight);
             ctx.fillStyle = '#FFF';
             ctx.fillRect(5, urlBarOffset + 5, urlBarWidth - 10, urlBarHeight - 10);
+
+            // Chris`s Loading Bar
```

Comment:
This rendering part looks great. 👍

Reply:
FYI I'm probably gonna remove this line, since git already attributes lines.

https://github.com/webmixedreality/exokit-home/blame/master/index.html